llvm/test/CodeGen/X86/llvm.asin.ll

```diff
@@ -1,63 +1,225 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 2
-; RUN: llc < %s -mtriple=x86_64-unknown-linux-gnu | FileCheck %s
```

Comment:
Most of the diff here is expanding the checks from x86_64 to also include i686, for both selectors. Better to precommit the additional target before adding the gisel runs